src/main/java/hello/Application.java

```diff
@@ -17,6 +17,11 @@ String generateSecretToken() {
 		return Long.toHexString(r.nextLong());
 	}
 
+    String generateCSRFToken() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Abbreviation in name 'generateCSRFToken' must contain no more than '4' consecutive capital letters. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.37.128.36:9000/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.37.128.36:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Class 'Application' looks like designed for extension (can be subclassed), but the method 'generateCSRFToken' does not have javadoc that explains how to do that safely. If class is not designed for extension consider making the class 'Application' final or making the method 'generateCSRFToken' static/final/abstract/empty, or adding allowed annotation for the method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.37.128.36:9000/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.design.DesignForExtensionCheck)